select-input.md

```diff
@@ -0,0 +1,424 @@
+<div align="center">
+
+# Select Input
+
+Search for and multi-select items in a dropdown.
+
+</div>
+
+
+## Create People
+
+Let's create a `people` table:
+
+```sh
+mix ecto.gen.migration add_people_table
```

Comment:
Any reason to not use [`mix phx.gen.schema`](https://hexdocs.pm/phoenix/Mix.Tasks.Phx.Gen.Schema.html) for this?

Reply:
Yes I could have use this command, I think I forgot it exists